pkg/ibmvpc/examples/out/explain_out/NACLQueryAllowSubset_all_vpcs_explain_debug.txt

```diff
@@ -1,7 +1,8 @@
-Connectivity explanation for protocol: UDP between vsi1-ky and 161.26.0.0/16 within test-vpc1-ky
-================================================================================================
+Explaining connectivity from vsi1-ky to 161.26.0.0/16 within test-vpc1-ky using "protocol: UDP"
+===============================================================================================
 
-Connection protocol: UDP src-ports: 1-600 dst-ports: 1-50 exists between vsi1-ky[10.240.10.4] and Public Internet 161.26.0.0/16 (note that not all queried protocols/ports are allowed)
+Connection from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16 using "protocol: UDP"
```

Comment:
Where did the ports go?

Reply:
The ports disappeared due to a typo mistake (I can always blame the late hour), but actually there is a real issue here.
The protocol - UDP here - is part of the **query** while the port are not part of the query, but of the existing connection. This distinction was supposed to be clear before the latest changes in the output, and is now lost.

Reply:
The first header (the one with the `=======`) should define the query 
The second header should define the actual connection
Thus, using the same wording in both ("using xxxx") is not good.

Reply:
I have no problems with repeating the connection details, but maybe I misunderstood something.
My understanding:
* On the first header you say: "here is what you asked in your query".
* On the second header you say: "here is what is allowed/not allowed".

Reply:
your understanding is correct.
the issue is not with repeating the details, but with making the distinction between the query and the actual connection. In the current version this distinction is not clear.